File: okio/src/jvmMain/java/okio/internal/Logger.java
package okio.internal;

public class Logger {
  private final java.util.logging.Logger logger;
  private Logger(String name) {
    this.logger = java.util.logging.Logger.getLogger(name);
  }
  public static Logger getLogger(String name) {
    return new Logger(name);
  }
  public void warn(String msg, Throwable e) {
    logger.log(java.util.logging.Level.WARNING, msg, e);
  }
}

File: okio/src/jvmMain/kotlin/okio/internal/Logger.kt
package okio.internal

internal class Logger internal constructor(name: String) {
  private val logger: java.util.logging.Logger = java.util.logging.Logger.getLogger(name)
  fun warn(msg: String, e: Throwable): Unit = logger.log(java.util.logging.Level.WARNING, msg, e)
}
